tox_DEBIAN/install.py

```diff
@@ -31,7 +31,13 @@ def install_debian_deps(path, deps, opts, action):
         tmp_usr = path_join(tmp_dir, 'usr')
         if isdir(tmp_usr):
             action.setactivity('copy', ', '.join(__iter_files(tmp_usr)))
-            action.popen(['cp', '-r'] + listdir(tmp_usr) + [path], cwd=tmp_usr)
+            _dirs = listdir(tmp_usr)
+            if 'py2' in path:
```

Comment:
I might be wrong here, but I'm wondering if this reliable enough. Maybe we could use `action.venv` to find out if it's a Python 2 environment or ask the Python interpreter inside the env for its version or something. What do you think?